CRM/Member/Form.php

```diff
@@ -453,9 +453,9 @@ protected static function getPriceSetID($params) {
     $priceSetID = CRM_Utils_Array::value('price_set_id', $params);
```

Comment:
@eileenmcnaughton You'll need a rebase.   this is now `$priceSetID = $params['price_set_id'] ?? NULL;`

Reply:
It doesn't seem stale yet below?

Reply:
(I can fix after if need be)

Reply:
I could not apply the patch without editing on master.    I am surprised this PR does not show that it has conflicts - could be me.